src/test/java/com/bunq/sdk/context/UserContextTest.java

```diff
@@ -0,0 +1,19 @@
+package com.bunq.sdk.context;
+
+import com.bunq.sdk.BunqSdkTestBase;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class UserContextTest extends BunqSdkTestBase {
+    
+    @Test
+    public void testConstruct() {
+        ApiContext context = getApiContext();
+
+        UserContext sut = new UserContext(context);
```

Comment:
what does `sut` stand for ? Why not just `userContext` ?

Reply:
bump

Reply:
sut = system-under-test, mostly to specify the thing you are testing